dropwizard-example/src/main/java/com/example/helloworld/resources/HelloWorldResource.java

```diff
@@ -39,10 +39,14 @@ public Saying sayHello(@QueryParam("name") Optional<String> name) {
     public void receiveHello(@Valid Saying saying) {
         LOGGER.info("Received a saying: {}", saying);
     }
-    
-    @GET
+
+    @POST
     @Path("/date")
-    public void receiveDate(@QueryParam("date") DateTimeParam dateTimeParam) {
-        LOGGER.info("Received a saying: {}", dateTimeParam.get());
+    public void receiveDate(@QueryParam("value") Optional<DateTimeParam> dateTimeParam) {
```

Comment:
Why did you change the query parameter name? "date" was more specific than the very generic "value".